zebra-consensus/src/block/check.rs

```diff
@@ -121,17 +121,9 @@ pub fn subsidy_is_valid(block: &Block, network: Network) -> Result<(), BlockErro
     } else if halving_div.count_ones() != 1 {
         unreachable!("invalid halving divisor: the halving divisor must be a non-zero power of two")
     } else if height < canopy_activation_height {
-        // Founders rewards are paid up to Canopy activation, on both mainnet and testnet
-        let founders_reward = subsidy::founders_reward::founders_reward(height, network)
-            .expect("invalid Amount: founders reward should be valid");
-        let matching_values = subsidy::general::find_output_with_amount(coinbase, founders_reward);
-
-        // TODO: the exact founders reward value must be sent as a single output to the correct address
-        if !matching_values.is_empty() {
-            Ok(())
-        } else {
-            Err(SubsidyError::FoundersRewardNotFound)?
-        }
+        // Founders rewards are paid up to Canopy activation, on both mainnet and testnet.
+        // But we checkpoint in Canopy so founders reward does not apply for Zebra.
+        Ok(())
```

Comment:
Maybe keeping the if arm here is not needed and just use:

```
     } else if halving_div < 4 and height >=0  {
```
in the arm below ?

I didn't make that for clarity but it should be ok to do it as well.